ksql-common/src/main/java/io/confluent/ksql/util/StringUtil.java

```diff
@@ -15,34 +15,25 @@
 
 package io.confluent.ksql.util;
 
-import java.util.List;
+import io.confluent.ksql.schema.ksql.FormatOptions;
 
 public final class StringUtil {
 
-  private StringUtil() {
-  }
+  private StringUtil() { }
```

Comment:
The way the file was is inline with our coding style.  This change will just get reverted by anyone who formats the file. 

I'd revert this.

```suggestion
  private StringUtil() {
  }
```

Reply:
😢 okay